ui/packages/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -289,12 +289,13 @@ define([
             }));
         },
         setupTextInput: function () {
-            // if the anyText filter is equal to a wildcard value then use a
-            // blank value as the text input value below
-            var isAnyTextWildcard = this.filter.anyText !== undefined && this.filter.anyText[0].value === '*';
+            // if the anyText filter is undefined or equal to a wildcard value then use a blank
+            // value as the text input value below
```

Comment:
❓I'm not sure what the behavior should be here. If a user adds a wildcard in the input box (with or without other spatial/contextual criteria), saves the search, then opens the search edit view, the wildcard will have been removed, right. Although searches will work, it might be a little unexpected to the user that the wildcard was removed.

Reply:
I changed it so it'll save the text entered in the input for both the basic and advanced forms